src/main/scala/nl.knaw.dans.easy.bag2deposit/ddm/DdmTransformer.scala

```diff
@@ -48,21 +48,31 @@ class DdmTransformer(cfgDir: File, collectionsMap: Map[String, Elem] = Map.empty
     relationRewriteRule,
   )
 
-  private def dcmiMetadataStandardRuleTransformer(newDcmiNodes: NodeSeq, profileTitle: String) = new RuleTransformer(
+  private def standardRuleTransformer(newDcmiNodes: NodeSeq, profileTitle: String) = new RuleTransformer(
     NewDcmiNodesRewriteRule(newDcmiNodes),
     DistinctTitlesRewriteRule(profileTitle),
     relationRewriteRule,
     languageRewriteRule,
+    ProfileDateRewriteRule,
   )
 
   private case class ArchaeologyRewriteRule(profileTitle: String, additionalDcmiNodes: NodeSeq) extends RewriteRule {
     override def transform(node: Node): Seq[Node] = {
       // TODO apply NewDcmiNodesRewriteRule/DistinctTitlesRewriteRule instead
-      if (node.label != "dcmiMetadata") node
-      else <dcmiMetadata>
-             { distinctTitles(profileTitle, dcmiMetadataArchaeologyRuleTransformer(node).nonEmptyChildren) }
-             { additionalDcmiNodes }
-           </dcmiMetadata>.copy(prefix = node.prefix, attributes = node.attributes, scope = node.scope)
+      if (node.label == "profile") {
```

Comment:
Less brackets and indentations to match with my eyes with the following approach

    node.label match {
      case "profile" -> ....
      case "dcmiMetadata" -> ....
      case _ -> node
    }